4.3/Dockerfile

```diff
@@ -4,7 +4,8 @@ STOPSIGNAL SIGINT
 
 RUN	addgroup -S adminer \
 &&	adduser -S -G adminer adminer \
-&&	mkdir -p /var/www/html
+&&	mkdir -p /var/www/html \
+&&	chown -R adminer:adminer /var/www/html
```

Comment:
The `entrypoint.sh` runs as `admin`. Without this, the user has no permission to create a symlink.